src/redux/actions/componentConfig/resetPlotConfig.js

```diff
@@ -0,0 +1,46 @@
+import { RESET_CONFIG, SAVE_CONFIG } from 'redux/actionTypes/componentConfig';
+import { initialPlotConfigStates } from 'redux/reducers/componentConfig/initialState';
+import fetchAPI from 'utils/http/fetchAPI';
+import handleError from 'utils/http/handleError';
+import endUserMessages from 'utils/endUserMessages';
+
+const resetPlotConfig = (experimentId, plotUuid, plotType) => async (dispatch) => {
+  const defaultConfig = initialPlotConfigStates[plotType];
+
+  try {
+    await fetchAPI(
+      `/v1/experiments/${experimentId}/plots-tables/${plotUuid}`,
+      {
+        method: 'PUT',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ config: defaultConfig }),
+      },
+    );
+
+    dispatch({
+      type: RESET_CONFIG,
+      payload: {
+        plotUuid,
+        config: defaultConfig,
+      },
+    });
```

Comment:
Just use `RESET_CONFIG` and add what you want of `SAVE_CONFIG` in the reducer for this